src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/service/SnsService.kt

```diff
@@ -45,19 +43,14 @@ class SnsService(hmppsQueueService: HmppsQueueService, private val objectMapper:
   }
 
   private fun publishToDomainEventsTopic(payload: HMPPSDomainEvent) {
-    log.debug("Event {} for id {}", payload.eventType, payload.additionalInformation)
-    domaineventsTopicClient.publish(
-      PublishRequest.builder()
-        .topicArn(domaineventsTopic.arn)
-        .message(objectMapper.writeValueAsString(payload))
-        .messageAttributes(
-          mapOf(
-            "eventType" to MessageAttributeValue.builder().dataType("String").stringValue(payload.eventType).build(),
-          ),
-        )
-        .build()
-        .also { log.info("Published event $payload to outbound topic") },
-    )
+    if (payload.eventType != null) {
+      log.debug("Event {} for id {}", payload.eventType, payload.additionalInformation)
+      domaineventsTopic.publish(
```

Comment:
I see that you tackled https://dsdmoj.atlassian.net/browse/MAP-1644 at the same time ;-)